src/components/Transactions.tsx

```diff
@@ -56,6 +57,8 @@ export const Transactions: FC<TransactionsProps> = () => {
   const [filteredCategories, setFilteredCategories] = useState([]);
   const [categoryType, setCategoryType] = useState(0);
   const transactionNotes = useRef('');
+  const [activeCategory, setActiveCategory] = useState([]);
```

Comment:
do we actually need this active category? maybe better use `transactionCategory` ?